### PR TITLE
Add minimal crate info

### PR DESCRIPTION
**Pull request by sophie-h**
Also adds minimum required version of the library to every crate doc.

### CODE REVIEW

File: atk/src/lib.rs [hunk 1]
```diff
@@ -1,10 +1,11 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
-//! # ATK bindings
+//! # Rust ATK bindings
 //!
-//! This library contains safe Rust bindings for [ATK](https://developer.gnome.org/atk/). It's
-//! a part of [Gtk-rs](https://gtk-rs.org/).
-
+//! This library contains safe Rust bindings for [ATK](https://developer.gnome.org/atk).
+//! It is a part of [Gtk-rs](https://gtk-rs.org/).
```
Comment (bilelmoussaoui) [inline atk/src/lib.rs:6]: Maybe it should mention it's part of gtk3-rs bindings instead of gtk-rs?

File: gdk/src/lib.rs [hunk 2]
```diff
@@ -4,6 +4,13 @@
 #![allow(clippy::missing_safety_doc)]
 #![allow(clippy::upper_case_acronyms)]
 
+//! # Rust GDK 3 bindings
```
Comment (GuillaumeGomez) [inline gdk/src/lib.rs:7]: I think it needs to be before the other attributes.

Comment (GuillaumeGomez) [reply gdk/src/lib.rs:7]: Ah no, nevermind!

**Review by GuillaumeGomez — APPROVED** [review]

File: atk/src/lib.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply atk/src/lib.rs:6]: We are talking about the organization there, not the repository (hence the URL).

Comment (bilelmoussaoui) [reply atk/src/lib.rs:6]: sure, what i meant is it's maybe worth it to mention that atk is part of the specific gtk3-rs bindings, which are part of gtk-rs.org.

Comment (GuillaumeGomez) [reply atk/src/lib.rs:6]: Not sure if this is worth it to mention it in the documentation. What do you think @sophie-h @sdroege ?

Comment (bilelmoussaoui) [reply atk/src/lib.rs:6]: then what mentioning that atk is part of gtk-rs.org brings to the docs? you will already be reading them from gtk-rs.org/gtk3-rs/ 
but it's just a suggestion, probably doesn't worth the change, feel free to ignore it

Comment (sophie-h) [reply atk/src/lib.rs:6]: I think there is too much room for improvement for all these missing introductory docs to start tweaking them now. Just wanted to have two sentences and a header everywhere.

Comment (bilelmoussaoui) [reply atk/src/lib.rs:6]: Indeed, never mind my comment :-)

**Review by bilelmoussaoui — APPROVED** [review]

**Pull request merged**